src/init_game.cpp

```diff
@@ -861,108 +708,109 @@ void deinitGame()
 		safePacketsReceivedMap[c].clear();
 	}
 #ifdef SOUND
-#ifdef USE_OPENAL
+#ifdef USE_OPENAL //TODO: OpenAL is now all of the broken...
```

Comment:
Not sure if we can just reuse the hack below...might need to rearchitect this area.